GOVERNANCE.md

```diff
@@ -1,47 +1,45 @@
-# Metrictank Governance
+# Governance
 
-This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Metrictank community. Common terminology used in this governance document are listed below:
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Metrictank community. Common terminology used in this governance document are listed below: 
 
-* **Team Members**: Any members [listed][members] in the project
+* **Team members**: Any members of the private [team mailing list][team].
 
-A loose sense of voting terminology (they are [defined](#voting) in greater detail later in this document):
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`][maintainers]).
 
-* **Lazy Consensus (Vote)**: Abstaining means agreement
-* **Majority Vote**: More than half vote in favor
-* **Supermajority Vote**: More than two thirds vote in favor
+* **Projects**: A single repository in the Grafana GitHub organization and listed below is referred to as a project:
+  * metrictank
+
+* **The Metrictank project**: The sum of all activities performed under this governance, concerning one or more repositories or the community.
 
 ## Values
 
-The Metrictank developers and community are expected to follow the values defined in the [Metrictank Code of Conduct][coc]. Furthermore, the Metrictank community strives for kindness, giving feedback effectively, and building a welcoming environment. The Metrictank Team generally decide by [lazy consensus](#consensus) and only resort to conflict resolution by a [majority](#majority-vote) vote if consensus cannot be reached.
+The Metrictank developers and community are expected to follow the values defined in the [Code of Conduct][coc]. Furthermore, the Metrictank community strives for kindness, giving feedback effectively, and building a welcoming environment. The Metrictank developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
```

Comment:
consensus means we need a list of who the developers are. is developers the same as maintainers?

Reply:
Yes; this is clear in the consensus & voting sections which define how to actually do things. Valid point, but I would prefer to not change it, or change it across all projects after a vote has passed either way.